### PR TITLE
Add apply_only argument. Add force_conflicts option.

### PR DESCRIPTION
**Pull request by kahirokunn**
https://docs.aws.amazon.com/eks/latest/userguide/add-user-role.html

Some cloud providers have resources such as `aws-auth` configmap that can be applied but never deleted permanently.
It would be smart to handle such cases as well.

### CODE REVIEW

File: kubernetes/resource_kubectl_manifest.go [hunk 1]
```diff
@@ -83,6 +83,9 @@ func resourceKubectlManifest() *schema.Resource {
 			return nil
 		},
 		DeleteContext: func(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+			if d.Get("apply_only").(bool) {
+				return nil
+			}
```
Comment (gavinbunney) [inline kubernetes/resource_kubectl_manifest.go:88]: Can you move this into the `resourceKubectlManifestDelete` function below?

**Comment by stevehipwell** [comment]
@kahirokunn what about also being able to define the field manager name?

**Comment by aidanmelen** [comment]
@gavinbunney when do you think this will get released to terraform registry?

File: docs/resources/kubectl_manifest.md [hunk 2]
```diff
@@ -42,6 +42,8 @@ YAML
 * `sensitive_fields` - Optional. List of fields (dot-syntax) which are sensitive and should be obfuscated in output. Defaults to `["data"]` for Secrets.
 * `force_new` - Optional. Forces delete & create of resources if the `yaml_body` changes. Default `false`.
 * `server_side_apply` - Optional. Allow using server-side-apply method. Default `false`.
+* `force_conflicts` - Optional. Allow using force_conflicts. Default `false`.
+* `apply_only` - Optional. It does not delete resource in any case Default `false`.
```
Comment (goetzc) [inline docs/resources/kubectl_manifest.md:46]: `s/ Default/. Default/`

**Pull request merged**